core/templates/pages/blog-dashboard-page/blog-dashboard-page.component.html

```diff
@@ -1,29 +1,48 @@
 <div *ngIf="activeTab === 'main'">
   <mat-card *ngIf="blogDashboardData" class="oppia-page-card blog-dashboard-card">
-    <mat-card-header class="blog-dashboard-header">
+    <mat-card-header class="blog-dashboard-header mr-0">
       <img mat-card-avatar
            [src]="authorProfilePictureUrl"
            alt="Profile image of the author of the blog post"
            class="e2e-test-profile-current-user-photo">
-      <mat-card-title class="blog-dashboard-card-title e2e-test-username-visible">
-        {{ blogDashboardData.username }}
-      </mat-card-title>
-      <mat-card-subtitle>
+      <mat-card-title class="w-100 d-flex justify-content-between">
+        <span class="blog-dashboard-card-title e2e-test-username-visible">
+          {{ authorName }}
+          <button class="fas fa-pen oppia-editor-edit-icon"
+                  title="Edit Author Name"
+                  (click)="showAuthorDetailsEditor()">
+            &#xE254;
```

Comment:
I think we're [replacing material icons with font-awesome equivalents](https://github.com/oppia/oppia/issues/15968). Since you are already using a font-awesome class above, is this line necessary?

Reply:
Removed.